packages/lib/src/toolbar/MeasuredControl.tsx

```diff
@@ -20,7 +20,9 @@ function MeasuredControl(props: Props) {
         }
       }}
     >
-      {({ measureRef }) => (
+      {(
+        { measureRef }, // eslint-disable-line @typescript-eslint/unbound-method
```

Comment:
This is because the library declares `measureRef` as a method:

```ts
measureRef(/* params */): void;

// instead of:
measureRef: (/* params */) => void;
```